### ISSUE TITLE
Script does not accept spaces in output directory name

### ISSUE THREAD

**Issue by marcin-bo** [contentful/contentful-utilities.swift#8]
Hi,

If I set output argument to a path that includes directory with spaces then I get error (`Illegal instruction: 4`) and script does not work:

```
$ contentful-utilities sync-to-bundle --space-id "X" --access-token "Y" --output "/Users/Z/Downloads/Directory With Spaces/Products"
Writing sync JSON files to directory /Users/Z/Downloads/Directory With Spaces/Products
Illegal instruction: 4
```

If I use  a path that does not includes directory with spaces then it works:
```
$ contentful-utilities sync-to-bundle --space-id "X" --access-token "Y" --output "/Users/Z/Downloads/DirectoryWithNoSpaces/Products"
Writing sync JSON files to directory /Users/Z/Downloads/DirectoryWithNoSpaces/Products
... // script downloads data
```

Maybe that's again the issue with Commander?

**Comment by matelo** [comment]
currently in PR: https://github.com/contentful/contentful-utilities.swift/pull/10

**Comment by jroehl** [comment]
Closed with #10
